connector/src/test/scala/com/basho/riak/spark/rdd/ReadFromRiakRDDTest.scala

```diff
@@ -17,29 +17,39 @@
  */
 package com.basho.riak.spark.rdd
 
+import java.io.Serializable
 import java.math.BigInteger
+import java.sql.Timestamp
+import java.time.Instant
 import java.util.UUID
+import com.basho.riak.client.api.convert.{Converter, ConverterFactory, JSONConverter}
+import com.basho.riak.client.core.util.BinaryValue
+import com.fasterxml.jackson.module.scala.DefaultScalaModule
 import org.junit.Assert._
 import org.junit.Test
 import org.junit.experimental.categories.Category
 import com.basho.riak.spark._
 
 case class UserTS(timestamp: String, user_id: String)
+//class UserTSTimestamp(val timestamp: Long) extends Serializable
 
 @Category(Array(classOf[IntegrationTests]))
 class ReadFromRiakRDDTest extends AbstractRDDTest{
+
+  JSONConverter.registerJacksonModule(DefaultScalaModule)
```

Comment:
I think this initialization should be somewhere in the Spark Connector already. Could you check it? From the users perspective it's better do not take care about such things..